docs/contracts/interfaces.rst

```diff
@@ -37,10 +37,32 @@ they can be overridden. This does not automatically mean that an overriding func
 can be overridden again - this is only possible if the overriding
 function is marked ``virtual``.
 
+Interfaces can inherit from other interfaces. This, in general, has the same rules as
+for normal contracts, except that an interface does not need to override a function
+that is inherited from two of its parents:
+
+::
+
+    pragma solidity >=0.6.2 <0.7.0;
```

Comment:
This is causing a failure, since we have not reached 0.6.2 yet. You can use `>0.6.1`.

Reply:
Fixed.